be/src/exec/tablet_sink_index_channel.cpp

```diff
@@ -809,6 +799,25 @@ Status NodeChannel::close_wait(RuntimeState* state) {
     // 2. wait eos request finish
     RETURN_IF_ERROR(_wait_all_prev_request());
 
+    // assign tablet dict infos
+    if (!_tablet_commit_infos.empty()) {
+        std::vector<std::string> invalid_dict_cache_columns;
+        invalid_dict_cache_columns.assign(_valid_dict_cache_info.invalid_dict_cache_column_set.begin(),
+                                          _valid_dict_cache_info.invalid_dict_cache_column_set.end());
+        _tablet_commit_infos[0].__set_invalid_dict_cache_columns(invalid_dict_cache_columns);
+
+        std::vector<std::string> valid_dict_cache_columns;
+        std::vector<int64_t> valid_dict_collected_versions;
```

Comment:
why the version is different?

Reply:
Some BEs don't have this field during grayscale upgrades, and we need to detect this case